test/functional/testlib.bash

```diff
@@ -407,9 +407,10 @@ set_env_variables() { # swupd_function
 	# different options for swupd
```

Comment:
you also need to update this function ->https://github.com/clearlinux/swupd-client/blob/master/test/functional/testlib.bash#L2985 since you added a new test group called "SIG"

Reply:
thanks. I'll update